### DISCUSSION TITLE
[src/output/render-manager.cpp:343] output test failed !

### DISCUSSION THREAD

**Discussion by Marietto2008** [WayfireWM/wayfire#2191] [General]
Hello.

I've reinstalled wayland / wayfire 0.9 from git on my main system. I'm using the nvidia driver version 535.146.02 and it can be launched,so it is not needed to install the nvidia driver 550. Anyway,actually I'm trying to fix a new incisive bug :

`[src/output/render-manager.cpp:343] output test failed !`

when I press CTRL + F1 I see a lot of those cycling and flooding messages ; all these messages make the system anymore responsive and an hard reset is required. They prevent some programs installed on the system to work properly,like Firefox. I don't know if this error is related to another error that I see and that I'm not able to fix :

`set class "me" resource limit memorylocked : operation not permitted.`

I see these errors even on the old system running with the nvidia driver 550,so they are not related with it. 

I have modified /etc/devfs.rules as follows :

```
[system=10]
add path 'usb/*' mode 0660 group operator
add path 'dri/*' mode 0666 group video
add path 'drm/*' mode 0666 group video
add path 'dri/*' mode 0666 group operator
add path 'drm/*' mode 0666 group operator
```

For sure,also :

```
# pw groupmod video -m marietto
# pw groupmod wheel -m marietto
# pw groupmod operator -m marietto
```

and :

```
# nano /home/marietto/.login_conf

me:\
  :charset=UTF-8:\
  :lang=it_IT.UTF-8:\
  :setenv=LC_COLLATE=C:\
  :memorylocked=unlimited:

# cap_mkdb /home/marietto/.login_conf
```

This message is also intriguing :

```
nvidia-modeset: Loading NVIDIA Kernel Mode Setting Driver for UNIX platforms  535.146.02  Sun Dec  3 13:40:40 UTC 2023
sysctl_warn_reuse: can't re-use a leaf (hw.dri.debug)!
```

**Comment by Marietto2008** [comment]
@soreau ; @ammen99

**Reply by ammen99** [reply]
@Marietto2008 I'm looking at your latest log from the other issue, it is again a permission problem, you'll have to fix the permissions and you usually do - you have posted snippets of commands in previous discussions.

**Reply by Marietto2008** [reply]
This problem arised some days after having fixed the permissions. It happened suddenly and I don't see what I might have did to cause it. It is visible on the old FreeBSD 14 installation and even on the new one,where I tried to replicate the procedure. This means that it is there while the permissions are fixed.

**Comment by Marietto2008** [comment]
We have fixed this bug. This is how :

```
[root@marietto /var/run/user]==> rsync -avxHAX 1001 1001_

sending incremental file list
1001/
1001/ICEauthority
1001/at-spi/
1001/at-spi/bus_0
1001/dbus-1/
1001/dbus-1/services/
1001/dconf/
1001/dconf/user
1001/gvfsd/
1001/pulse/
1001/pulse/native
1001/pulse/pid

sent 747 bytes  received 107 bytes  1.708,00 bytes/sec
total size is 170  speedup is 0,20

[root@marietto /var/run/user]==> chmod 700 1001_
[root@marietto /var/run/user]==> chown -R marietto:wheel 1001_
[root@marietto /var/run/user]==> sudo chmod 700 /var/run/user/1001_


wayfire-start :

#!/usr/local/bin/zsh
export LD_LIBRARY_PATH=/opt/wayfire/lib/$libdir
export PATH=/opt/wayfire/bin:$PATH
export XDG_RUNTIME_DIR=/var/run/user/1001_
export PKG_CONFIG_PATH=/opt/wayfire/share/pkgconfig:/opt/wayfire/libdata/pkgconfig
env WLR_NO_HARDWARE_CURSORS=1 ck-launch-session dbus-run-session wayfire

nano /home/marietto/.zshenv

export MOZ_ENABLE_WAYLAND=1
export XDG_RUNTIME_DIR=/var/run/user/1001_
export MOZ_DISABLE_WAYLAND_PROXY=1
```

The parameter "LIBSEAT_BACKEND=consolekit2" may work on Linux but it's not good under FreeBSD.
